src/cmd/troubleshoot/checks_executor.go

```diff
@@ -0,0 +1,48 @@
+package troubleshoot
+
+import (
+	tserrors "github.com/Dynatrace/dynatrace-operator/src/cmd/troubleshoot/errors"
+)
+
+type Result int
+
+const (
+	PASSED = iota + 1
+	FAILED
+	SKIPPED
+)
+
+var results = map[string]Result{}
```

Comment:
global `map` is prime real estate for memory leaks, refactor either into parameter or a struct. Maybe a something like a `CheckSuit` struct is viable containing the array of checks as well as any result. `runChecks` could be a method of that.

Reply:
It's not an issue here because of `trobuleshoot` command nature - single run of sequential checks with single `results` instance in the end. 
I didn't want it over-engineered.

Reply:
as far as I can tell it's populated in `runChecks` and read in `skipCheck`. defining and instantiating it in the former and passing it to the latter doesn't seem over-engineered to me